docker/jetsontx2/Dockerfile.tx2

```diff
@@ -0,0 +1,231 @@
+FROM ubuntu:16.04
+MAINTAINER jeremiah.liou@icuro.com
+
+# ICURO Santa Clara, CA
+# Authors: Jeremiah Liou, Arun Sunil, Cody Heiner
+
+############################################################
+################### Install JetPack ########################
+############################################################
+
+CMD ["/bin/bash"]
+
+WORKDIR /root
+
+RUN /bin/bash -c " \
+   # Install Dependencies
+      apt-get update; \
+      apt-get install -y sudo wget tar bzip2; \
+   # Install Drivers
+      cd /root; \
+      wget https://developer.download.nvidia.com/devzone/devcenter/mobile/jetpack_l4t/3.2.1/m8u2ki/JetPackL4T_321_b23/Tegra186_Linux_R28.2.1_aarch64.tbz2; \
+      tar -xvjf Tegra186_Linux_R28.2.1_aarch64.tbz2; \
+      cd Linux_for_Tegra; bash apply_binaries.sh -r /; \
+      cd ..; \
+      rm -rf Linux_for_Tegra; \
+      rm Tegra186_Linux_R28.2.1_aarch64.tbz2; \
+   # Install CUDA
+      cd /root; \
+      wget https://developer.download.nvidia.com/devzone/devcenter/mobile/jetpack_l4t/3.2.1/m8u2ki/JetPackL4T_321_b23/cuda-repo-l4t-9-0-local_9.0.252-1_arm64.deb; \
+      dpkg -i cuda-repo-l4t-9-0-local_9.0.252-1_arm64.deb; \
+      apt-key add /var/cuda-repo-9-0-local/7fa2af80.pub; \
+      apt-get update; \
+      apt-get install -y cuda-toolkit-9.0; \
+      rm cuda-repo-l4t-9-0-local_9.0.252-1_arm64.deb; \
+   # Install cuDNN
+      cd /root; \
+      wget https://developer.download.nvidia.com/devzone/devcenter/mobile/jetpack_l4t/3.2.1/m8u2ki/JetPackL4T_321_b23/libcudnn7_7.0.5.15-1+cuda9.0_arm64.deb; \
+      dpkg -i libcudnn7_7.0.5.15-1+cuda9.0_arm64.deb; \
+      wget https://developer.download.nvidia.com/devzone/devcenter/mobile/jetpack_l4t/3.2.1/m8u2ki/JetPackL4T_321_b23/libcudnn7-dev_7.0.5.15-1+cuda9.0_arm64.deb; \
+      dpkg -i libcudnn7-dev_7.0.5.15-1+cuda9.0_arm64.deb; \
```

Comment:
@gbiggs according to [this](https://docs.nvidia.com/deeplearning/sdk/cudnn-sla/index.html#distribution)
I think its ok to include cuDNN in binary form right?

Reply:
I'm not sure, but possibly not. The license from NVIDIA is non-transferable. Short of getting a lawyer, @esteve, any opinions?